superdesk/core/resources/service.py

```diff
@@ -390,8 +390,8 @@ async def create(self, _docs: Sequence[ResourceModelType | dict[str, Any]]) -> L
             if self.config.versioning:
                 await self.insert_versioned_document(doc_dict)
 
-        await self.on_created(docs)
-        return ids
+        await self.on_created(instances)
+        return instances
```

Comment:
This change will probably break `newsroom-core` as the returned value used to be a list of integers and we're using it there to fetch the newly created item from db.

Reply:
I will create a separate PR to address this